modules/d/tests/test_gmake.lua

```diff
@@ -105,6 +105,11 @@ endif
 		test.capture [[
 $(OBJDIR)/blah.o: blah.d
 	@echo $(notdir $<)
+ifeq (posix,$(SHELLTYPE))
+	$(SILENT) mkdir -p $(OBJDIR)
+else
+	$(SILENT) mkdir $(subst /,\\,$(OBJDIR))
+endif
```

Comment:
This seems to be an unrelated change? I think this is how it used to be but it was causing issues when running multiple jobs.